plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/view/dev/Styles.js

```diff
@@ -75,15 +78,27 @@ Ext.define('NX.view.dev.Styles', {
         'Other'
       ];
 
-      Ext.each(sections, function (section) {
-        var component = Ext.create('NX.view.dev.styles.' + section);
-        me.add(component);
+      me.logDebug('Creating style guide');
+
+      // TODO: See if suspending layouts here actually helps anything?
+      //Ext.AbstractComponent.suspendLayouts();
+      //try {
```

Comment:
FTR I'm leaving this in as reminder on how to properly suppsend/resume layouts... we may need to apply this to other parts that do this.  Unsure however if this does anything at all for this specific bit of code.